examples/connector/python/main.py

```diff
@@ -9,27 +9,56 @@
 from sdk_pb2 import connector_sdk_pb2
 
 
-class ConnectorService(connector_sdk_pb2_grpc.ConnectorServicer):
+class ConnectorService(connector_sdk_pb2_grpc.SourceConnectorServicer):
     def ConfigurationForm(self, request, context):
-        form_fields = common_pb2.ConfigurationFormResponse(schema_selection_supported=True,
+        response = common_pb2.ConfigurationFormResponse(schema_selection_supported=True,
                                                            table_selection_supported=True)
-        form_fields.fields.add(name="apiKey", label="API Key", required=True, text_field=common_pb2.TextField.PlainText)
-        form_fields.fields.add(name="password", label="User Password", required=True, text_field=common_pb2.TextField.Password)
-
-        form_fields.fields.add(
-            name="region",
-            label="AWS Region",
-            required=False,
-            dropdown_field=common_pb2.DropdownField(dropdown_field=["US-EAST", "US-WEST"])
+        response.fields.add().single.CopyFrom(
```

Comment:
instead of using CopyForm every time, we can use kwargs to initialise the message, something like
```
        response.fields.add(field=common_pb2.Field(
            name="apiKey",
            label="API Key",
            required=True,
            text_field=common_pb2.TextField.PlainText
        ))
```

Reply:
We cannot directly add the field as FormField takes either `FieldSet` or `single`.

Reply:
we can supply `single=common_pb2.Field(...` to the add method